tests/acceptance/features/apiSharingNg/sharedWithMe.feature

```diff
@@ -2974,4 +2974,444 @@ Feature: an user gets the resources shared to them
+                            "type": "string",
+                            "pattern": "^%permissions_id_pattern%$"
+                          },
+                          "invitation": {
+                            "type": "object",
+                            "required": ["invitedBy"],
+                            "properties": {
+                              "invitedBy": {
+                                "type": "object",
+                                "required": ["user"],
+                                "properties": {
+                                  "user": {
+                                    "type": "object",
+                                    "required": ["displayName", "id"],
+                                    "properties": {
+                                      "displayName": {
+                                        "type": "string",
+                                        "enum": ["Alice Hansen"]
+                                      },
+                                      "id": {
+                                        "type": "string",
+                                        "pattern": "^%user_id_pattern%$"
+                                      }
+                                    }
+                                  }
+                                }
+                              }
+                            }
+                          },
+                          "roles": {
+                            "type": "array",
+                            "items": [
+                              {
+                                "type": "string",
+                                "pattern": "^%role_id_pattern%$"
+                              }
+                            ]
+                          }
+                        }
+                      ]
+                    },
+                    "size": {
+                      "type": "number",
+                      "enum": [8]
+                    }
+                  }
+                },
+                "size": {
+                  "type": "number",
+                  "enum": [8]
+                }
+              }
+            }
+          }
+        }
+      }
+      """
+
+
+  Scenario: group member lists the space folder shared with them when auto-sync is disabled
+    Given using spaces DAV path
+    And user "Carol" has been created with default attributes and without skeleton files
+    And group "grp1" has been created
+    And the following users have been added to the following groups
+      | username | groupname |
+      | Brian    | grp1      |
+      | Carol    | grp1      |
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "projectSpace" with the default quota using the Graph API
+    And user "Alice" has created a folder "folderToShare" in space "projectSpace"
+    And user "Brian" has disabled the auto-sync share
```

Comment:
if brain set auto-sync disable in all scenario then we can shift this step to background